config/betamocks/services_config.yml

```diff
@@ -464,3 +464,16 @@
     - :method: :get
       :path: "/server/rest/services/VA/FacilitySitePoint_VHA/FeatureServer/0"
       :file_path: "gis/vha/metadata"
+
+#PSSG
+- :name: 'PSSG'
+  :base_uri: <%= "#{URI(Settings.locators.drive_time_band_base_path).host}:#{URI(Settings.locators.drive_time_band_base_path).port}" %>
+  :endpoints:
+    - :method: :get
+      :path: "/arcgis2/rest/services/Portal/MonthlyVAST_TTB/FeatureServer/0/*"
+      :file_path: "pssg/drive_time_bands" 
```

Comment:
Figured out why I couldn't get a match! This should be `pssg/drivetime_bands` to match up with the structure in the `vets-api-mockdata` repo.

Reply:
d'oh